lib/bots/versionbot.ts

```diff
@@ -390,6 +409,228 @@ export class VersionBot extends ProcBot {
     }
 
     /**
+     * Checks a freshly opened PR to see if there are any configured reviewers and maintainers.
+     * Should any valid reviewers and maintainers not already be assigned as reviewers on the PR, then
+     * a comment is posted on the PR directly asking those user logins to add themselves.
+     *
+     * @param _registration GithubRegistration object used to register the method
+     * @param event         ServiceEvent containing the event information ('pull_request' event)
+     * @returns             A void Promise once execution has finished.
+     */
+    protected addReviewers = (_registration: GithubRegistration, event: ServiceEvent): Promise<void> => {
+        const pr = event.cookedEvent.data.pull_request;
+        const head = event.cookedEvent.data.pull_request.head;
+        const owner = head.repo.owner.login;
+        const repo = head.repo.name;
+        let approvedMaintainers: string[];
+        let approvedReviewers: string[];
+
+        // Only when opening a new PR.
+        console.log(event.cookedEvent.data.action);
+        if (event.cookedEvent.data.action !== 'opened') {
```

Comment:
If the GH webhooks are running slow, and VB never gets the `opened` event, is there a way it could still perform this task?